src/script/components/list/ConversationListCell.tsx

```diff
@@ -42,9 +42,9 @@ export interface ConversationListCellProps {
   conversation: Conversation;
   dataUieName?: string;
   index: number;
-  is_selected: (conversation: Conversation) => boolean;
```

Comment:
camelCase?

Reply:
Ahaha fair point :) 
But since this is a parameter that is already there and can be instantiated in ko templates, I won't have any typecheck there and I have to admit I'd rather not take that risk :upside_down_face: